notebooks/en/labelling_feedback_setfit.ipynb

```diff
@@ -0,0 +1,2874 @@
+{
```

Comment:
<p>The image is the same as earlier. Perhaps, we should remove the one in the introduction to avoid duplication? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>